### PR TITLE
Update system.md by adding NTP details

### PR DESCRIPTION
**Pull request by axkar**
NTP functionality has been thoroughly verified and the proper documentation is added on:
- Configuring NTP
- Verifying NTP Status
- Verifying NTP Sources

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Test changes
  - [ ] Checked in changed Readme.adoc (make test-spec)
  - [ ] Added new test to group Readme.adoc and yaml file
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [x] Documentation content changes
  - [ ] ChangeLog updated (for major changes)
- [ ] Other (please describe):

### CODE REVIEW

File: doc/system.md [hunk 1]
```diff
@@ -201,6 +201,78 @@ admin@host:/>
...
+||                                \     |          |  zzzz = estimated error.
+||                                 |    |           \
+MS Name/IP address         Stratum Poll Reach LastRx Last sample               
+===============================================================================
+^* 192.36.143.134                1   6   177     9   +278ms[ -3845s] +/-  514ms
+admin@infix-00-00-00:/>
+```
+This command provides a detailed view of the current NTP sources, including their reachability, 
+poll interval, and offset.
+
```
Comment (troglobit) [inline doc/system.md:275]: This subsection is perhaps the most suited to reveal to the user what implementation we're using under the hood.  So at the end you could add something like this:

> The system uses `chronyd` for Network Time Protocol (NTP) synchronization, the output shown here is best explained in the [Chrony documentation](https://chrony-project.org/doc/4.6.1/chronyc.html).

The styling with a "qoutation" formatting is always highlighted by Markdown engines in one way or another.  The GitHub variant is one, others indent it, change color, or even put a bounding box around it.  So it's very useful for this type of information.

**Review by troglobit — APPROVED** [review]
A few comments, nothing major, because overall it looks really good!

**Review by troglobit — CHANGES_REQUESTED** [review]
(Managed to click "approve" again instead of "request changes")

File: doc/system.md [hunk 2]
```diff
@@ -201,6 +201,78 @@ admin@host:/>
...
+Below is an example configuration for enabling NTP with a specific server
+and the `iburst` option for faster initial synchronization.
+
+```
+admin@host:/> configure
+admin@host:/config/> set system ntp enabled
+admin@host:/config/> set system ntp server ntp-pool
+admin@host:/config/> set system ntp server ntp-pool udp address pool.ntp.org
+admin@host:/config/> set system ntp server ntp-pool iburst
+admin@host:/config/> set system ntp server ntp-pool prefer
```
[the remark this answers is not in the document]
Comment (axkar) [reply doc/system.md:216]: I think I was consistent per file :)
admin@host has been used in system.md file
admin@example we have in networking.md

[the remark this answers is not in the document]
Comment (axkar) [reply doc/system.md:216]: There is actually only one place in system.md where we use admin@example but that should be changed to admin@host for consistensy:
Inside "Changing Hostname" section...

File: doc/system.md (hunk 1 above)
Comment (axkar) [reply doc/system.md:275]: Thnx

File: doc/system.md [hunk 3]
```diff
@@ -201,6 +201,78 @@ admin@host:/>
...
+
+This configuration enables the NTP client and sets the NTP server to `pool.ntp.org` 
+with the `iburst` and `prefer` options. The `iburst` option ensures faster initial 
+synchronization, and the `prefer` option designates this server as preferred.
+
+`prefer false`: The NTP client will choose the best available source based on several factors, 
+such as network delay, stratum, and other metrics (default config).
+
+`prefer true`: The NTP client will try to use the preferred server as the primary source unless 
+it becomes unreachable or unusable.
```
[the remark this answers is not in the document]
Comment (axkar) [reply doc/system.md:227]: Fixed

**Comment by axkar** [comment]
> A few comments, nothing major, because overall it looks really good!

Thnx for all comment. Hope I have not forgotten anything :)

File: doc/system.md [hunk 4]
```diff
@@ -201,6 +201,82 @@ admin@host:/>
...
+MS Name/IP address         Stratum Poll Reach LastRx Last sample               
+===============================================================================
+^* 192.36.143.134                1   6   177     9   +278ms[ -3845s] +/-  514ms
+admin@host:/>
+```
+
+> The system uses `chronyd` for Network Time Protocol (NTP)
+synchronization. The output shown here is best explained in the
+[Chrony documentation](https://chrony-project.org/doc/4.6.1/chronyc.html).
+
```
Comment (troglobit) [inline doc/system.md:279]: For multi-line quotes, you usually need to prefix each line with `> `, but it seems the GitHub Markdown engine (gfm), somehow clues this together itself.  Look here, your branch:

 - <https://github.com/kernelkit/infix/blob/add-docs-ntp/doc/system.md#show-ntp-sources>

I'm very surprised by this, but for consistency, and compatibility with other Markdown engines, I think it's best to make sure each line in the quote is prefixed with `> `.

**Review by troglobit — CHANGES_REQUESTED** [review]
Great work on this, almost there! :smiley:

**Review by troglobit — APPROVED** [review]
Great stuff, thanks for hanging in there! :tada: 

For future *documentation only changes*, consider adding the `[skip ci]` text to the body of your commit message(s), since there's no point in occupying the build and test servers.

**Comment by axkar** [comment]
> Great stuff, thanks for hanging in there! 🎉
> 
> For future _documentation only changes_, consider adding the `[skip ci]` text to the body of your commit message(s), since there's no point in occupying the build and test servers.

Yes, I'll do that :) Thnx

**Pull request merged**